packages/clerk-js/package.json

```diff
@@ -35,8 +35,8 @@
     "prepublishOnly": "npm run build",
     "postpublish": "node ./scripts/purge-cache.mjs",
     "start": "echo \"Noop\"",
-    "test": "jest",
-    "test:coverage": "jest --collectCoverage && open coverage/lcov-report/index.html"
+    "test": "TZ=UTC jest",
```

Comment:
Makes sense, but is there any other way to do this?

Reply:
Yes, we can extend the `jsdom` environment to set `process.env.TZ` before setup. I will refactor.